pkg/aggregate/aggregate.go

```diff
@@ -129,6 +129,25 @@ func RawAggregatesToStringList(aggregates string) ([]string, error) {
 	return list, nil
 }
 
+func ParseCrossLabelSets(str string) [][]string {
+	var res [][]string
+	labelSetStrings := strings.Split(str, ".")
```

Comment:
need to consider the dot here with another delimiter...

Reply:
Agreed to use semicolon instead of dot.